bip-musig2.mediawiki

```diff
@@ -282,7 +307,7 @@ Input:
 * If ''len(sk) > 0'':
 ** Let ''rand'' be the byte-wise xor of ''sk'' and ''hash<sub>MuSig/aux</sub>(rand')''<ref>The random data is hashed (with a unique tag) as a precaution against situations where the randomness may be correlated with the secret signing key itself. It is xored with the secret key (rather than combined with it in a hash) to reduce the number of operations exposed to the actual secret key.</ref>.
 * Else: let ''rand = rand' ''
-* Let ''k<sub>i</sub> = int(hash<sub>MuSig/nonce</sub>(rand || bytes(1, len(aggpk)) || aggpk || bytes(1, i) || bytes(1, len(m)) || m || bytes(4, len(extra_in)) || extra_in)) mod n'' for ''i = 1,2''
+* Let ''k<sub>i</sub> = int(hash<sub>MuSig/nonce</sub>(rand || bytes(1, len(aggpk)) || aggpk || bytes(1, len(m)) || m || bytes(4, len(extra_in)) || extra_in || bytes(1, i))) mod n'' for ''i = 1,2''
```

Comment:
This is only in this PR because I based this PR on another PR. This particular change has already gone away because the corresponding PR got merged and this PR was rebased.